content/enterprise_influxdb/v1.9/administration/backup-and-restore.md

```diff
@@ -1,13 +1,13 @@
 ---
-title: Back up and restore InfluxDB Enterprise clusters
+title: Back up and restore
 description: >
   Back up and restore InfluxDB enterprise clusters in case of unexpected data loss.
```

Comment:
"In case of" sounds like loss has already happened. Is "to prevent" or "to avoid" better?